BlueShift-iOS-SDK/BlueshiftLog.h

```diff
@@ -0,0 +1,18 @@
+//
+//  BlueshiftLog.h
+//  BlueShift-iOS-SDK
+//
+//  Created by Ketan Shikhare on 29/07/20.
+//  Copyright © 2020 Bullfinch Software. All rights reserved.
```

Comment:
must be a copy-paste error :)

Reply:
This gets created automatically on file creation, maybe the organization is set as Bullfinch in SDK xcode project. I will fix this.